ui/src/app/pages/profile/profile-page.tsx

```diff
@@ -152,6 +161,49 @@ interface ProfilePageState {
   updating: boolean;
 }
 
+interface CompletionTime {
+  completionTime: number;
+  bypassTime: number;
+}
+
+const getRegistrationStatus = (completionTime: number, bypassTime: number) => {
+  return completionTime !== null && completionTime !== undefined ? RegistrationStepStatus.COMPLETED :
+  bypassTime !== null && completionTime !== undefined ? RegistrationStepStatus.BYPASSED : RegistrationStepStatus.UNCOMPLETE;
+};
+
+const bypassedText = (bypassTime: number): JSX.Element => {
+  return <React.Fragment>
+  <div>Bypassed on:</div>
+  <div>{displayDateWithoutHours(bypassTime)}</div>
+</React.Fragment>;
+};
+
```

Comment:
modified for better re-use (with lens). Also pulled from class and made into a function